packages/blocks/src/api/raw-handling/paste-handler.js

```diff
@@ -24,6 +24,7 @@ import shortcodeConverter from './shortcode-converter';
 import markdownConverter from './markdown-converter';
 import iframeRemover from './iframe-remover';
 import googleDocsUIDRemover from './google-docs-uid-remover';
+import HTMLFormattingRemover from './html-formatting-remover';
```

Comment:
Nit: Naming should be capitalized as `htmlFormattingRemover`

>If an abbreviation or an acronym occurs at the start of a variable name, it must be written to respect the camelcase naming rules covering the first letter of a variable or class definition. **For variable assignment, this means writing the abbreviation entirely as lowercase.** For class definitions, its initial letter should be capitalized.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#abbreviations-and-acronyms

Reply:
Done.